jon-jc/academia
# Add Analytics Chart to Pricing Page

# Add Analytics Chart to Pricing Page

## Description
This PR adds a new analytics chart feature to the pricing page, showcasing the data visualization capabilities of Academia Premium. It implements both line and bar charts using the `recharts` library, integrated into our existing UI components.

## Changes
- Added a new "Powerful Analytics at Your Fingertips" section to the pricing page
- Implemented a card component with a tabs interface for switching between line and bar charts
- Created sample data for citations, downloads, and readers over a 12-month period
- Ensured responsiveness and dark mode compatibility for the charts
- Added necessary accessibility features

## Screenshots
[Insert screenshots of the new chart section in both light and dark mode]

## Testing Instructions
1. Navigate to the pricing page
2. Scroll to the "Powerful Analytics at Your Fingertips" section
3. Verify that both line and bar charts are displayed correctly
4. Test tab switching between line and bar charts
5. Check responsiveness by resizing the browser window
6. Toggle dark mode and ensure charts adapt correctly
7. Use keyboard navigation to interact with the charts and verify accessibility

## Checklist
- [x] Components are properly typed
- [x] Accessibility features are implemented
- [ ] Dark mode compatibility is ensured
- [ ] Responsive design is implemented

## Related Issue
Closes #[Insert issue number here]

## Additional Notes
This new feature significantly enhances the visual appeal of our pricing page and provides a tangible demonstration of Academia Premium's analytics capabilities.
